### PR TITLE
Add Sort event feature and Display feature on Developer Guide

### PR DESCRIPTION
**Pull request by solkim-83**
Have added the following sections on our developer guide:
1. Sort event feature (Could possibly be merged w/ Bangyi's sort contact feature write-up under one "Sort" feature heading)
2. Display feature (List event feature was written)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -285,6 +291,7 @@ Action | Format, Examples
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Delete Event** | `deleteEvent INDEX`<br> e.g., `deleteEvent 2`
+**Delete Event** | `deleteEvent INDEX`<br> e.g., `deleteEvent 2`
```
Comment (LinkedInk) [inline docs/UserGuide.md]: Duplicate

**Review by LinkedInk — COMMENTED** [review]
lgtm mostly except that one duplicate line in UG's command summary.

File: docs/UserGuide.md (hunk 1 above)
Comment (solkim-83) [reply docs/UserGuide.md]: Thanks for pointing it out. Have made the edit.

**Review by LinkedInk — APPROVED** [review]
Update the .png files as well.

Otherwise its ok to merge.

**Comment by solkim-83** [comment]
Have updated the .png images hence merging them.

**Pull request merged**